src/main/webapp/availability.js

```diff
@@ -59,7 +59,11 @@ function updateAvailability() {
   };
   let requestBody = JSON.stringify(requestObject);
   let request = new Request('/availability', {method: 'PUT', body: requestBody});
-  fetch(request).then(() => location.reload()).catch((error) => {
+  fetch(request).then(() => {
+      if (buttonType === 'updateButton') {
```

Comment:
Some docu explaining why this check would be good

Reply:
Done